File: JavaNIDE/volume/app/src/main/java/sample/For9.java
package sample;
import java.util.*;

class For9
 {
   static int i,j,k,fact=1,fact1=1,n;
   static float sum=0,fact2=0;
  public static void main(String[] ar)
 {
    Scanner t=new Scanner(System.in);
    System.out.println("enter no");
    n=t.nextInt();
    for(i=1;i<=n;i++)
    {
      System.out.print(i+"!"+"/");
     for(j=1;j<=i;j++)
     {
     fact=fact*j;
     System.out.print(j+"!");
     if(j<i)
     {
      System.out.print("+");
     }
    for(k=1;k<=j;k++)
   {
    fact1=fact1*k;
   }
   fact2=fact2+fact1;
   fact1=1;
   }
   System.out.print(" + ");
   sum=sum+fact/fact2;
    fact2=0;
    fact=1;
     }
     System.out.print("sum of seriese="+sum);
     }
    }

File: CodingC/for 9.c
#include<stdio.h>
  int main()
 {
   float m,n,x,r,p;
   float sum=0;
   printf("enter the no");
   scanf("%f",&n);
   for(x=1;x<=n-1;x++)
{
   m=(x+1);
  printf("%.0f*%.0f/((%.0f*%.0f)-((%.0f*%.0f)/(%.0f*%.0f)-(%.0f*%.0f)))",x,x,x,x,m,m,x,x,m,m);
  printf("    +    ");
  p=x*x;
  r=p/(p-(((m*m)/(p-(m*m)))));
  sum=sum+r;
 }
 printf("your sum of seriese is %.2f",sum);
return 0;
}
  
  